#Clear the Display
bool clearDisplay

#Position inside the display where the text should appear
uint8 line
uint8 pos

#String to print at given position
string data
